Mathlib/MeasureTheory/Group/MeasurableEquiv.lean

```diff
@@ -201,3 +203,29 @@ def divLeft [MeasurableMul G] [MeasurableInv G] (g : G) : G ≃ᵐ G where
   measurable_invFun := measurable_inv.mul_const g
 
 end MeasurableEquiv
+
+namespace MeasureTheory.Measure
+variable {G A : Type*} [Group G] [AddCommGroup A] [DistribMulAction G A] [MeasurableSpace A]
+  [MeasurableSpace G] -- not needed actually
```

Comment:
If it's not needed, then why is it here? I guess there is a good reason (related to the comment of the next line?) and you just need to improve the comment.